conf/prod.routes

```diff
@@ -1,5 +1,5 @@
 # Add all the application routes to the app.routes file
-->         /transits-movements-trader-at-departure-stub              app.Routes
+->         /common-transit-convention-trader-at-departure              app.Routes
```

Comment:
Shouldn't the context here be the same as the repo name?  Same as https://github.com/hmrc/transit-movements-trader-at-destination/blob/master/conf/prod.routes

Reply:
> Shouldn't the context here be the same as the repo name? Same as https://github.com/hmrc/transit-movements-trader-at-destination/blob/master/conf/prod.routes

Currently this is correct as the path is being used like this when called from the departures backend. I can change this to better indicate that it's the stub